src/config/wallets.ts

```diff
@@ -36,6 +37,7 @@ export const WalletModalOption = {
   SubWalletEvm: SupportWallet.SubWalletEvm,
   Metadot: SupportWallet.Metadot,
   Wallet3: SupportWallet.Wallet3,
+  OneKey: SupportWallet.OneKey,
 };
 
 export const SubstrateWallets = [
```

Comment:
It looks like Onekey wallet supports Substrate wallets, you can add Onekey here and `supportWalletObj`